ragen/env/deepcoder/env.py

```
@@ -0,0 +1,187 @@
+
+    def _load_data(self) -> None:
+        dataset_path = self.config.dataset_path
+        if dataset_path:
+            self.dataset = load_dataset(path=dataset_path, cache_dir=self.config.cache_dir)
+            return
+
+        dataset_bundle = None
+        try:
+            train_dataset, test_dataset = prepare_deepcoder_data()
+            dataset_bundle = {"train": train_dataset, "test": test_dataset}
+        except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)